source/blog/2020-02-27-fall-winter.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: 🍂 Fall/Winter ❄️ 2019 Quarterly Review
+published: true
+post_author:
+  display_name: Ian Bennett
+  twitter: enbnt
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+
+Hi Finaglers,
+
+
+
+It is ⌛time to take a 👀 look back on the 🌇 twilight of the year 2019, when the seasons shift from 🍂 Fall to Winter ❄️. Across our projects, we have introduced support for JDK 11 and added support for Scala 2.13 across a large portion of our projects. A major thanks due to our OSS community [contributions](https://github.com/twitter/finagle/issues/771) on continuing to drive Scala 2.13 support - we couldn’t have made such progress so quickly without your help. We have made some steady progress and improvements across all of our projects, but also want to loop you in on some of our plans looking forward to Spring 🌱.
+
+### Finatra
+
+We have introduced a new Case Class Validation library (finatra/validation), upgraded some dependent libraries, squashed some bugs, and moved some things around. Automatic handling of Mustache rendering has been removed from finatra/http and Mustache support has been broken into two separate libraries: finatra/mustache and finatra/http-mustache. Please realize that we try to minimize breaking API changes unless they’re absolutely necessary and we apologize for the inconvenience when upgrading. Much of the movement is to clean up inconsistencies or poor layout of existing APIs and in service to some bigger things coming.
+
+You should expect to see improvements to the usability of our new Case Class Validation library, bug fixes and broader support for Jackson annotations within finatra-jackson, and the removal of finagle-http as a dependency within finatra-jackson as we move into Spring 🌱. We are also looking into shifting Finatra’s HTTP path DSL to be based off of the OpenAPI spec and more information will follow as those plans solidify.
+
+We have continued to refine the APIs around c.t.util Streaming types (Reader, Writer, Pipe). We improved the performance of Reader. We have removed the deprecated response.StreamingResponse. [StreamingResponse and StreamingRequest](https://twitter.github.io/finatra/user-guide/http/streaming.html) are the recommended Finatra Streaming APIs.
+
+### Finagle
+
+[Martijnhoekstra](https://github.com/martijnhoekstra) added a [new HeaderMap](https://github.com/twitter/finagle/pull/805) to the finagle-http project!We madethis implementation the [default HeaderMap implementation](https://github.com/twitter/finagle/commit/597e78159db35eee9c5353d15243ab0d2889767d), resulting in improved performance - most notably in HTTP and HTTP/2 servers.
```

Comment:
`!We madethis implementation` --> `! We made this implementation` (spacing is off)